packages/schema/docs/build/schema.md

```diff
@@ -1053,12 +1053,12 @@ How will we get a list of new objects? Will be turned into a trigger automatical
 
 * ```
   { display: { label: 'New User', description: 'Trigger when a new User is created in your account.' },
-    operation: 
+    operation:
```

Comment:
The trailing spaces were re-added in my last PR. I don't know why they keep coming back. 🤷‍♂ Maybe your IDE automatically strips trailing spaces for all files in the repo? Mine only does that to a file when it's saved.